### PR TITLE
fix: Improve accuracy of log record timestamps

### PR DESCRIPTION
**Pull request by CyshallChan**
- Modify calculation of log record timestamp in "otlp.go" file to include nanoseconds

#1031

### CODE REVIEW

**Review by messixukejia — APPROVED** [review]

File: pkg/protocol/converter/otlp.go [hunk 1]
```diff
@@ -82,7 +82,7 @@ func (c *Converter) ConvertToOtlpResourseLogs(logGroup *protocol.LogGroup, targe
 		}
 
 		logRecord.SetObservedTimestamp(pcommon.Timestamp(time.Now().UnixNano()))
```
Comment (messixukejia) [inline pkg/protocol/converter/otlp.go:84]: ut失败了，需要修复下

Comment (CyshallChan) [reply pkg/protocol/converter/otlp.go:85]: @messixukejia 已修复

File: pkg/protocol/converter/otlp_test.go [hunk 2]
```diff
@@ -73,6 +76,66 @@ func TestNewConvertToOtlpLogs(t *testing.T) {
...
+				convey.So(5, convey.ShouldEqual, resourceLogs.Resource().Attributes().Len())
+
+				convey.Convey("Then the LogRecords should be valid", func() {
+					for i := 0; i < resourceLogs.ScopeLogs().Len(); i++ {
+						scope := resourceLogs.ScopeLogs().At(i)
+						convey.So(2, convey.ShouldEqual, scope.LogRecords().Len())
+
+						for j := 0; j < scope.LogRecords().Len(); j++ {
+							logRecord := scope.LogRecords().At(j)
+							convey.So(logs[i].Contents[4].Value, convey.ShouldEqual, logRecord.Body().AsString())
```
Comment (Abingcbc) [inline pkg/protocol/converter/otlp_test.go:132]: 单测这里没有验证两个时间字段有没有被正确设置

**Review by Abingcbc — CHANGES_REQUESTED** [review]

File: pkg/protocol/converter/otlp_test.go (hunk 2 above)
Comment (CyshallChan) [reply pkg/protocol/converter/otlp_test.go:136]: 已经加上了，看最新提交

**Review by Abingcbc — APPROVED** [review]

**Pull request merged**